prowler/providers/aws/services/elbv2/elbv2_service.py

```diff
@@ -51,6 +61,48 @@ def __describe_load_balancers__(self, regional_client):
                 f"{regional_client.region} -- {error.__class__.__name__}[{error.__traceback__.tb_lineno}]: {error}"
             )
 
+    def __describe_target_groups__(self, regional_client):
+        logger.info("ELBv2 - Describing target groups...")
+        try:
+            for lb in self.loadbalancersv2:
+                try:
+                    describe_target_groups_paginator = regional_client.get_paginator(
+                        "describe_target_groups"
+                    )
+                    for page in describe_target_groups_paginator.paginate(
+                        LoadBalancerArn=lb.arn
+                    ):
+                        for target_group in page["TargetGroups"]:
+                            for target_health in regional_client.describe_target_health(
+                                TargetGroupArn=target_group["TargetGroupArn"]
+                            )["TargetHealthDescriptions"]:
+                                tg = TargetGroups(
+                                    name=target_group["TargetGroupName"],
+                                    arn=target_group["TargetGroupArn"],
+                                    target_type=target_group["TargetType"],
+                                    target=target_health["Target"]["Id"],
```

Comment:
It is a best practice to use `get` function in dictionaries instead of keys because if an key is not required it could make fail the creation of the object. Look at this [doc](https://diveintopython.org/functions/dictionary-methods/get) to watch how to use it and define default values in case of fail.

Reply:
Not sure if this is needed since no other files are using .get to access values from dictionaries from the boto3 api. I think it looks better if I just follow what others have done. But if you strongly insist I change it, I can do that.